ProfilerCore/Brofiler.h

```diff
@@ -1,5 +1,13 @@
 #pragma once
 
+#include "Types.h"
+
+#ifdef LINUX64
+#include "Linux/PublicCommonLinux.h"
+#else
+#include "Windows/PublicCommonWindows.h"

```

Comment:
I think its much better to have a layout like: 

linux/os/PublicCommon.h
windows/os/PublicCommon.h

Depending on the os, you include linux or windows, and all you do is include <os/PublicCommon.h>